[HANDLOWY] New View EmployeeSalesDepartment

AdriannaArtych opened the pull request.

onta1 committed:
Dodanie okna technolgii i możliwości dodawanie nowych rekordów

onta1 committed:
Dodanie funkcji Edytuj i Usuń oraz Powrót

onta1 committed:
Uporządkowanie kodu

DamianSzupienko committed:
[Production] Added code for creating records in Production Process table, based on records from Planned Production table with value "InProduction" set to "true"

DamianSzupienko committed:
[Production] Code for creating records - applied requested changes

JustynaKras committed:
[HANDLOWY] Changes in OutsourcingForm

JustynaKras committed:
[HANDLOWY] Changes in AddNewOutsourcingCompanyForm

DamianSzupienko committed:
[Production] Updated and fixed SQL and EF model

DamianSzupienko committed:
[Production] Conflict resolved

onta1 committed:
Conflicts resolved, changed Timepermeter to TimePerMeter

AdriannaArtych committed:
[HANDLOWY] sql update

AdriannaArtych committed:
[HANDLOWY] sql + model update in VS

AdriannaArtych committed:
[HANDLOWY] requested changes

marcinwat committed:
size of buttons are now equals

YupiKayey committed:
Add DevelopmentForm

DamianSzupienko committed:
[Production] Fixed and updated EF model, resolved conflicts

michalmolski committed:
handling exceptions

michalmolski committed:
edit existing plan

michalmolski committed:
not necessary buttons removed

michalmolski committed:
bug fixed

DamianSzupienko committed:
Merge branch 'master' into Feature/Production-Technology

PatrykBanach95 committed:
Added DevelopmentForm to QualityControlForm and programmed return and main menu buttons

lotysb committed:
Merge branch 'master' into feature/HANDLOWY/OutsourcingForm

marcinwat committed:
some changes in edit form gui

AdriannaArtych committed:
[HANDLOWY] rozwiązanie konfliktu

marcinwat committed:
general product forms gui improvement

marcinwat committed:
Merge branch 'master' of https://github.com/rudnickirobert/KWZP2019 into feature/production/products_form_buttons

marcinwat committed:
latest gui changes

AdriannaArtych committed:
[HANDLOWY] new employee

rudnickirobert committed:
Merge pull request #62 from rudnickirobert/Feature/Production_process/Code_for_adding_records

[Production] Added code for creating records in Production Process ta…

rudnickirobert committed:
Merge pull request #80 from rudnickirobert/feature/production/products_form_buttons

[Production] products_GUI

rudnickirobert committed:
Merge pull request #82 from rudnickirobert/feature/production/planned_production_handling_exceptions

[PRODUCTION] Planned Production handling exceptions

rudnickirobert commented on RoofingCompany.sql:
```diff
@@ -766,7 +766,7 @@ ON SafetyControl.IdInspectedEmployee = Employee.IdEmployee;
 
 --Views UR--
-
+/*
```
nie commitujemy zakomentowanego kodu- jeśli tworzenie widoku powoduje błędy w wykonaniu skryptu należy go naprawić (https://deviq.com/boy-scout-rule/)
... albo usunąć

DamianSzupienko committed:
[Production] Conflicts resolved, empty lines removed

michalmolski committed:
skipping weekends in date calculating process

michalmolski committed:
planned production form GUI standardized

JustynaKras committed:
[HANDLOWY] changes in OutsourcingForm and AddNewOutsourcingForm

michalmolski committed:
New planned production GUI standardized

rudnickirobert committed:
Merge pull request #65 from rudnickirobert/Feature/Production-Technology

[Production] Technology form

rudnickirobert committed:
Merge pull request #69 from rudnickirobert/feature/HANDLOWY/OutsourcingForm

Feature/HANDLOWY/outsourcing form

michalmolski committed:
not used method removed

minimilenka committed:
[Quality Control] modyfikacja insertów w bazie

PatrykBanach95 committed:
Merge branch 'master' into DevelopmentForm

PatrykBanach95 committed:
Corrected Entity Framework model

Bartelix committed:
Added:
- newTechnologyForm;
- editTechnologyForm

marcinwat committed:
data grid view style changed

marcinwat committed:
addproduct form - comboboxes and textboxes colours changed

marcinwat committed:
edit product form - changed colors in textBoxes and comboBoxes

AdriannaArtych committed:
Merge branch 'feature/handlowy/sql/update' of https://github.com/rudnickirobert/KWZP2019 into feature/handlowy/view/employee

AdriannaArtych committed:
[HANDLOWY] poprawki

AdriannaArtych committed:
[HANDLOWY] poprawione widoki

rudnickirobert committed:
Merge pull request #81 from rudnickirobert/DevelopmentForm

Add DevelopmentForm

rudnickirobert commented:
konflikty

rudnickirobert committed:
Merge pull request #90 from rudnickirobert/feature/QualityControl/sql_modified_inserts

[Quality Control] modyfikacja insertów w bazie

Bartelix committed:
deleted not working views from hr

rudnickirobert committed:
Merge pull request #96 from rudnickirobert/feature/production/products_gui

[Production] Feature/production/products gui

rudnickirobert committed:
Merge pull request #100 from rudnickirobert/feature/handlowy/sql/next/update

[HANDLOWY] poprawione widoki

michalmolski committed:
resolved
 Conflicts:
	KWZP2019/KWZP2019/Properties/Resources.Designer.cs
	KWZP2019/KWZP2019/Properties/Resources.resx

Bartelix committed:
Resolved conflicts

PatrykBanach95 committed:
[QualityControl] Changes in OSHControl Form and OSHControlHistory Form

AdriannaArtych committed:
[HANDLOWY] poprawione konflikty

AdriannaArtych committed:
[HANDLOWY] poprawione konflikty

rudnickirobert committed:
Merge pull request #74 from rudnickirobert/feature/handlowy/sql/update

[HANDLOWY] sql update

rudnickirobert reviewed (changes requested):
konflikty

rudnickirobert committed:
Merge pull request #87 from rudnickirobert/production/feature/skipping_weekend_days

[Production] skipping weekend days

rudnickirobert committed:
Merge pull request #95 from rudnickirobert/feature/ProductionTechnology

[Production] Technology: new and edit forms

rudnickirobert committed:
Merge pull request #103 from rudnickirobert/QualityControlCorrectedOSHControl

[QualityControl] Changes in OSHControl Form and OSHControlHistory Form

AdriannaArtych committed:
[HANDLOWY] poprawione konflikty

AdriannaArtych commented:
usunięty konflikt
